Demo-ObjC/ChannelViewController.m

```diff
@@ -323,9 +368,7 @@ - (void)keyboardWillShow:(NSNotification *)notification {
     
     self.keyboardAdjustmentConstraint.constant = keyboardHeight;
     [self.view setNeedsLayout];
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
-        [self scrollToBottomMessage];
-    });
+    [self scrollToBottomMessageWithDelay:0.1];
```

Comment:
Not a big deal but just curious about the delay amount. Is it trying to workaround UI something? Seeing 0.1 & 0.0 :p